src/lando/remote_settings.py

```diff
@@ -39,3 +39,12 @@
     "root": {"handlers": ["null"]},
     "disable_existing_loggers": True,
 }
+
+EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
```

Comment:
`EMAIL_BACKEND` is defined here and in `settings.py`. For consistency, you could remove one, or import to the required file if it should be defined in both places.

Reply:
This is an overridden from the value in `settings`, which is set to the console backend.

Reply:
Make sense, I didn't notice they are different. Not great that they have the same name and almost the exact same namespace, tbh.